src/query/expression/src/values.rs

```diff
@@ -523,6 +538,7 @@ impl PartialOrd for Scalar {
             (Scalar::Date(d1), Scalar::Date(d2)) => d1.partial_cmp(d2),
             (Scalar::Array(a1), Scalar::Array(a2)) => a1.partial_cmp(a2),
             (Scalar::Map(m1), Scalar::Map(m2)) => m1.partial_cmp(m2),
+            (Scalar::Bitmap(b1), Scalar::Bitmap(b2)) => b1.partial_cmp(b2),
```

Comment:
Direct comparison of binary values may produce unintended results, do we need to convert it to `RoaringBitmap` before comparing?